language/move-prover/spec-lang/src/translate.rs

```diff
@@ -2686,6 +2678,24 @@ impl<'env, 'translator> ModuleTranslator<'env, 'translator> {
             std::mem::take(&mut self.instantiation_map),
             std::mem::take(&mut self.spec_block_infos),
         );
+        // Warn about unused schemas.
```

Comment:
I moved it here so that module_env is available.

Reply:
Ah, SG then!